app/albums/create/page.tsx

```
@@ -6,28 +6,33 @@ import { useRouter } from "next/navigation";
 export default function CreatePage() {
 	const { currentUser } = useContext(authContext);
-	const uid = currentUser?.uid;
+	const userId = currentUser?.uid;
 	const dispatch = useDispatch<AppDispatch>();
 	const router = useRouter();
-	const [loading, setLoading] = useState(true);
+	const searchParams = useSearchParams();
+	const shareRoomId = searchParams.get("shareRoomId");
+	const sharedRoomTitle = searchParams.get("sharedRoomTitle");
 
 	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: userId

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)